### PR TITLE
 feat(TempestFansub): website changes

### PR DESCRIPTION
**Pull request by BySuspect**
## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

</details>

### CODE REVIEW

File: websites/T/Tempest Fansub/presence.ts [hunk 1]
```diff
@@ -1,38 +1,41 @@
 const presence = new Presence({
 		clientId: "1047102386478534727",
 	}),
-	browsingTimestamp = Math.floor(Date.now() / 1000);
+	browsingTimestamp = Math.floor(Date.now() / 1000),
+	defaultLogo =
+		"https://cdn.rcd.gg/PreMiD/websites/T/Tempest%20Fansub/assets/logo.png",
+	ecchilogo = "https://imgur.com/BtQLEdY.png";
```
Comment (DarkVillager) [inline websites/T/Tempest Fansub/presence.ts:7]: Please implement this with a const enum Assets.

```suggestion
	browsingTimestamp = Math.floor(Date.now() / 1000);
	
	const enum Assets {
	Logo = "https://cdn.rcd.gg/PreMiD/websites/T/Tempest%20Fansub/assets/logo.png",
	EcchiLogo = "https://imgur.com/BtQLEdY.png",
```

File: websites/T/Tempest Fansub/presence.ts [hunk 2]
```diff
@@ -1,38 +1,41 @@
...
 		searchParams = new URLSearchParams(location.search);
-	//manga domain changed 2 times so i added both
 	if (
-		hostname === "tempestscans.com" ||
-		hostname === "manga.tempestfansub.com"
+		hostname === "tempestfansub.com" ||
+		hostname === "ecchi.tempestfansub.com"
 	) {
 		//Manga-----------------------------------------------------------------------
+		if (hostname.startsWith("ecchi")) presenceData.largeImageKey = ecchilogo;
```
Comment (DarkVillager) [inline websites/T/Tempest Fansub/presence.ts:21]:
```suggestion
		if (hostname.startsWith("ecchi")) presenceData.largeImageKey = Assets.EcchiLogo;
```

File: websites/T/Tempest Fansub/presence.ts [hunk 3]
```diff
@@ -1,38 +1,41 @@
...
+		else presenceData.largeImageKey = defaultLogo;
```
Comment (DarkVillager) [inline websites/T/Tempest Fansub/presence.ts:22]:
```suggestion
		else presenceData.largeImageKey = Assets.Logo;
```

**Review by DarkVillager — CHANGES_REQUESTED** [review]

File: websites/T/Tempest Fansub/presence.ts [hunk 4]
```diff
@@ -2,37 +2,41 @@ const presence = new Presence({
 		clientId: "1047102386478534727",
 	}),
 	browsingTimestamp = Math.floor(Date.now() / 1000);
-
+const enum Assets {
+	Logo = "https://cdn.rcd.gg/PreMiD/websites/T/Tempest%20Fansub/assets/logo.png",
+	EcchiLogo = "https://imgur.com/BtQLEdY.png",
```
Comment (theusaf) [inline websites/T/Tempest Fansub/presence.ts:7]: Assets should be at least 512x512px

**Review by theusaf — CHANGES_REQUESTED** [review]

**Review by theusaf — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**